campaign-launcher/interface/src/hooks/human-protocol-sdk/escrow.ts

```diff
@@ -72,6 +72,8 @@ export const useCreateEscrow = () => {
 
       const escrowConfig = {
         ...oracles,
+        exchangeOracle:signer.address,
+        exchangeOracleFee:oracles.recordingOracleFee,
```

Comment:
Why you use the recording oracle fee for the exchange oracle? Keep the exchange oracle fee env variable.